settings.php

```diff
@@ -220,6 +220,23 @@
  */
 // $settings['omit_vary_cookie'] = TRUE;
 
+/**
+ * Expiration of cache_form entries:
+ *
+ * Backdrop's Form API stores details of forms in cache_form and these entries
+ * are kept for at least 6 hours by default. Expired entries are cleared by
+ * cron. Busy sites can encounter problems with the cache_form table becoming
+ * very large. It's possible to mitigate this by setting a shorter expiration
+ * for cached forms. In some cases it may be desirable to set a longer cache
+ * expiration. For example to prolong cache_form entries for Ajax forms in
+ * cached HTML.
+ *
+ * @see form_set_cache()
+ * @see system_cron()
+ * @see ajax_get_form()
+ */
+// $settings['form_cache_expiration'] = 21600;
```

Comment:
I don't think this will ever get called. Unlike in D7 where variable_get was overridden by settings here, I think with config_get we need to manually add a call to settings_get('form_cache_expiration') before, check if it returns anything. If not then use config_get.

Reply:
...fixed this by using `settings_get()` in `core/includes/form.inc` as per suggestion. Thanks 👍